hdr_schemata/models/HDRUK/3.0.0/schema.json

```diff
@@ -454,6 +453,9 @@
                               "title": "Patient pathway description"
                         }
                   },
+                  "required": [
+                        "spatial"
+                  ],
```

Comment:
do you know why this is coming in now? was it missed from a previous PR ?

really need to sort that out so this is generated automatically rather than by hand each time